File: leetcode/LeetCode_561_array.py
class Solution:
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        flag = True
        sum = 0
        for i in nums:
            if flag:
                sum += i
            flag = not flag
        return sum

File: leetcode/LeetCode_561_array.cpp
class Solution {
public:
    int arrayPairSum(vector<int>& nums) {
        std::stable_sort(nums.begin(), nums.end());
        int sum = 0;
        bool tmp = true;
        vector<int>::iterator i;
        for(i = nums.begin(); i != nums.end(); i++)
        {
            if(tmp)
                sum += *i;
            tmp = !tmp;
        }
        return sum;
    }
};
